src/web/mjs/connectors/Piccoma.mjs

```diff
@@ -0,0 +1,275 @@
+            , s = a.getContext("2d");
+        a.width = image.width,
+        a.height = image.height + global_n + global_r;
+        if (!pdata.isScrambled) {
+            s.drawImage(image, 0, 0, image.width, image.height, 0, 0, image.width, image.height);
+            return a;
+        }
+        var l = Math.ceil(image.width / num)
+            , h = (image.height,
+            function (t) {
+                var n = {};
+                return n.slices = t.length,
+                n.cols = function (t) {
+                    if (1 == t.length)
+                        return 1;
+                    for (var n = "init", r = 0; r < t.length; r++)
+                        if ("init" == n && (n = t[r].y),
+                        n != t[r].y)
+                            return r;
+                    return r;
+                }(t),
+                n.rows = t.length / n.cols,
+                n.width = t[0].width * n.cols,
+                n.height = t[0].height * n.rows,
+                n.x = t[0].x,
+                n.y = t[0].y,
+                n;
+            }
+            )
+            , p = function () {
+                var n, r = {};
+                for (n = 0; n < c; n++) {
+                    var e = {}
+                        , i = Math.floor(n / l)
+                        , u = n - i * l;
+                    e.x = u * num,
+                    e.y = i * num,
+                    e.width = num - (e.x + num <= image.width ? 0 : e.x + num - image.width),
+                    e.height = num - (e.y + num <= image.height ? 0 : e.y + num - image.height),
+                    r[e.width + "-" + e.height] || (r[e.width + "-" + e.height] = []),
+                    r[e.width + "-" + e.height].push(e);
+                }
+                return r;
+            }();
+        for (var v in p) {
+            var y, d = h(p[v]), g = [];
+            for (y = 0; y < p[v].length; y++)
+                g.push(y);
+            for (g = _shuffleSeed_(g, seed),
+            y = 0; y < p[v].length; y++) {
+                var b = g[y]
+                    , m = Math.floor(b / d.cols)
+                    , x = (b - m * d.cols) * p[v][y].width
+                    , S = m * p[v][y].height;
+                s.drawImage(image, d.x + x, d.y + S, p[v][y].width, p[v][y].height + global_e, p[v][y].x, p[v][y].y + global_n, p[v][y].width, p[v][y].height);
+            }
+        }
+        return a;
+    }
+}
+
+// from https://github.com/webcaetano/shuffle-seed
+function _shuffleSeed_(arr, seed) {
+    const size = arr.length;
+    const rng = _seedrandom_(seed);
+    const resp = [];
+    const keys = [];
+    for (let i = 0; i < size; i++) keys.push(i);
+    for (let i = 0; i < size; i++) {
+        const r = Math.floor(rng() * keys.length);
+        const g = keys[r];
+        keys.splice(r, 1);
+        resp.push(arr[g]);
+    }
+    return resp;
+}
+
+// from https://github.com/davidbau/seedrandom
+const _seedrandom_ = (function () {
```

Comment:
Is there a reason why this is wrapped in an immediately invoked function expression?
Seems to me like this can be removed